ajax/libs/thrift/package.json

```
@@ -0,0 +1,35 @@
+{
+  "name": "thrift",
+  "filename": "js/src/thrift.min.js",
```

```suggestion
  "filename": "js/src/thrift.js",
```

This file doesn't appear to exist in at least the most recent version: https://gitbox.apache.org/repos/asf?p=thrift.git;a=tree;f=lib/js/src;h=df5f7662763d9840df3522f8bccf897ef6897396;hb=cecee50308fc7e6f77f55b3fd906c1c6c471fa2f